Client/Frontend/Widgets/TwoLineCell.swift

```diff
@@ -205,7 +205,7 @@ class TwoLineHeaderFooterView: UITableViewHeaderFooterView {
 }
 
 private class TwoLineCellHelper {
-    var container: UIView!
+    weak var container: UIView!
```

Comment:
We should never be explicitly unwrapped a weak var since there is no guarantee that the ref will be held. I would change this to be an optional instead and update any call sites to either use guard checks or optional unwrapping.